application/core/LSGettextMoFile.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/*
+* LimeSurvey
+* Copyright (C) 2007-2023 The LimeSurvey Project Team / Carsten Schmitz
+* All rights reserved.
+* License: GNU/GPL License v2 or later, see LICENSE.php
+* LimeSurvey is free software. This version may have been modified pursuant
+* to the GNU General Public License, and as distributed it includes or
+* is derivative of works licensed under the GNU General Public License or
+* other free or open source software licenses.
+* See COPYRIGHT.php for copyright notices and details.
+*/
+
+/**
+ * Extends CGettextMoFile to make it compatible with PHP 8.1
+ */
+class LSGettextMoFile extends CGettextMoFile
```

Comment:
Hm this wasn't fixed in Yii 1.1 update?

Reply:
If I am not wrong, I think we did this before Yii had any support for PHP8.1.
I could be wrong though.

Any way, We should check, agree